noir-projects/aztec-nr/aztec/src/utils/field.nr

```diff
@@ -0,0 +1,124 @@
+global KNOWN_NON_RESIDUE: Field = 5; // This is a non-residue in Noir's native Field.
```

Comment:
Ultimately, this should all go in the stdlib Field dir. It's all to get a sqrt function.